test/integration/analysis.js

```diff
@@ -7,6 +7,8 @@ var Analysis = require('../../lib/analysis');
 var BatchClient = require('../../lib/postgresql/batch-client');
 
 var testConfig = require('../test-config');
+var testHelper = require('../helper');
+var DataObservatoryMultipleMeasures = require('../../lib/node/nodes/data-observatory-multiple-measures');
```

Comment:
Yes, I thought about that but the problem is that we check for the behavior when the analysis SQL is generated by the Analysis factory. We don't only depend on the node but also in the `waterfall` operations made by the factory which only receives a node definition. In our case, we create a node but it doesn't exist in the reference. Any ideas?